flang/lib/Lower/OpenMP/ClauseProcessor.h

```diff
@@ -226,7 +230,7 @@ bool ClauseProcessor::processMotionClauses(
               asFortran.str(), bounds, {},
               static_cast<
                   std::underlying_type_t<llvm::omp::OpenMPOffloadMappingFlags>>(
```

Comment:
Same thing about "using".  This should reduce the nesting level of the expression.